tidy3d/plugins/adjoint/web.py

```diff
@@ -123,6 +123,7 @@ def run(
         callback_url=callback_url,
         verbose=verbose,
     )
+    # TODO: add task_id
```

Comment:
I guess these TODO's are here because the PR only explicitly adds to `JaxSimulationData`? I think I'm missing some context here...

Reply:
the PR only adds to the JaxSimulationData when we have access to the `task_id` because we can't grab the task_id from a completed `web.run()` call, I have no way of getting it in these places unless we figure out some way around it. I think for the issue brought up by the user, it should be sufficient